src/scss/components/_serviceRichText.scss

```diff
@@ -0,0 +1,40 @@
+.richtext_title {
+  @include heading-4;
+
+  color: var(--brand-navy);
+  margin: 0;
+  padding-top: 6.4rem;
+}
+
+.richtext_container {
+  > h3 {
+    color: var(--brand-navy);
+    @include heading-5;
+
+    margin: 0;
+    padding-top: 1.6rem;
+  }
+
+  > p {
+    @include body-m-reg;
+
+    color: var(--grey-2);
+    margin: 0;
+    padding-top: 0.8rem;
+  }
+
```

Comment:
Please make sure to include ordered lists as well as per the user story requirements.

Reply:
done.